imap/message.c

```diff
@@ -190,6 +190,8 @@ EXPORTED int message_copy_strict(struct protstream *from, FILE *to,
     int munge8bit = config_getswitch(IMAPOPT_MUNGE8BIT);
     int inheader = 1, blankline = 1;
     struct buf tmp = BUF_INITIALIZER;
+    char is_from;
+    static const char * from_header = "From ";
```

Comment:
This shall be changed from *char to char[], so that the string is moved in readonly section and shared by all libimap callers.